api/pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "ttbud"
+version = "1.2"
+description = "Your virtual table friend"
+authors = ["Stephen Johnson <github@sjoh.net>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
```

Comment:
for python you probably want to specify an exact dependency

Reply:
also, specify python 3.8 because that's what you're using

Reply:
> for python you probably want to specify an exact dependency

After discussion and looking around, it looks like this is standard practice, the poetry tool itself uses this

Reply:
Changing to 3.8 sounds good, but still leaving out the patch number for now.